package.js

```diff
@@ -0,0 +1,17 @@
+Package.describe({
+  name: 'lodash:lodash',
+  version: '3.7.0',
```

Comment:
I noticed the `underscore` meteor package is maintained by the [mdg](https://atmospherejs.com/mdg) org and so isn't prefixed by a username unlike the [five lodash meteor packages](https://atmospherejs.com/mdg?q=lodash). Is there another way? \@glasser

I wonder if there's a way to get the official package supported in much the same way as `underscore` or